src/lib/sidenav/drawer.ts

```diff
@@ -325,6 +328,27 @@ export class MdDrawerContainer implements AfterContentInit {
   /** Event emitted when the drawer backdrop is clicked. */
   @Output() backdropClick = new EventEmitter<void>();
 
+  /**
+   * Width of the container at which it breaks
+   * e.g., breakpointWidth="500"
+   */
+  @Input() breakpointWidth: number;
```

Comment:
Coerce this? See https://github.com/angular/material2/blob/master/src/lib/slider/slider.ts#L149-L164